wasm.md

```diff
@@ -636,38 +690,59 @@ Currently, in the expanded form, the `export`s will come after the definition of
     syntax FuncExports ::= List{FuncExport, ""} [klabel(listFuncExport)]
 ```
 
+### Function Local Declaration
+
+```k
+    syntax LocalDecl      ::= "(" LocalDecl ")"     [bracket]
+                            | "local" ValTypes
+    syntax LocalDecls     ::= List{LocalDecl , ""} [klabel(listLocalDecl)]
+ // ----------------------------------------------------------------------
+
+    syntax VecType ::=  asLocalType ( LocalDecls            ) [function]
+                     | #asLocalType ( LocalDecls , ValTypes ) [function]
+ // -------------------------------------------------------------------
+    rule  asLocalType(LDECLS) => #asLocalType(LDECLS, .ValTypes)
+
+    rule #asLocalType(.LocalDecls            , VTYPES) => [ VTYPES ]
+    rule #asLocalType(LDECL:LocalDecl LDECLS , VTYPES) => #asLocalType(LDECLS, VTYPES) [owise]
```

Comment:
Can't you match on the parentheses here? And are we really supposed to throw away the `LDECL`? Why is it not necessary?

Reply:
whoops.. this rule may should not exist at all. Thanks for finding this out!